README.md

```diff
@@ -48,4 +48,24 @@ custom:
         value: ${opt:MyKey}
       - SomeOtherKeyThatAssignsRandomValue
 ```
+
+### Specifying encrypted key values
+
+First Use a KMS key in the region from command line to encrypt the key
+```
+  aws kms encrypt --key-id f7c59c6b-83de-4e80-8011-0fbd6846c695 --plaintext BzQ86PiX9t9UaAQsNWuFHN9oOkiyOwd9yXBu8RF1 | base64 --decode
+```
+
+Then configure the `value` as { encrypted: "AQICAHinIKhx8yV+y97+qS5naGEBUQrTP8RPE4HDnVvd0AzJ/wGF2tC0dPMHO..." }
+
+```yaml
+custom:
+  apiKeys:
```

Comment:
@suil - for v3.0.0 the configuration has changed a little (i have added stage variable).. can you please update this section of readme accordingly.

Reply:
changed. please review